paasta_tools/mesos_tools.py

```diff
@@ -104,6 +107,34 @@ def get_mesos_leader():
         raise ValueError('Expected to receive a valid URL, got: %s' % url)
 
 
+def get_mesos_proxy(cluster, location):
+    with open(DEFAULT_SERVICES_LOCATION, 'r') as istream:
+        services = yaml.load(istream)
+
+    region = None
+    for ltype in type_utils.available_location_types():
+        try:
+            region = type_utils.convert_location_type(
+                location,
+                ltype,
+                'region',
+            )[0]
```

Comment:
Add break after this line? If we have found the region already, then we don't have to iterate over rest of the regions?